projects/canopy/src/lib/header/header.component.spec.ts

```diff
@@ -400,4 +398,76 @@ describe('HeaderComponent', () => {
       });
     });
   });
+
+  describe('with account menu', () => {
+    let tabKeyDownEvent: KeyboardEvent;
+    let focusSpy: jasmine.Spy;
+    let preventDefaultSpy: jasmine.Spy;
+
+    beforeEach(() => {
+      fixture = MockRender(`
```

Comment:
Could you please update all the `MockRender` to have `reset: true`? 
```
MockRender(`<hmtl code>`, {}, { reset: true })
```

Otherwise we'll get all the warnings in the logs of the unit tests.

Reply:
Had to use the flush testbed method